StRoot/StEvent/StFcsCollection.h

```diff
@@ -64,9 +67,9 @@ class StFcsCollection : public StObject {
     StSPtrVecFcsPoint   mPoints[kFcsNDet];  
 
     Int_t mFcsReconstructionFlag=0;     // undefined for now
+    int mDataExist=0;
```

Comment:
As this is involved in IO, the preference is to use Int_t  instead of int.

Reply:
From the name of this variable, it appears it can be boolean

Reply:
It's single int per event.... I don't think we need this at the moment, but there are zero-suppressed data and raw-none-zero-suppressed data from DAQ. We only read zero-suppressed data at offline so far, unlike some online reader which reads raw. If we ever needed to read raw, we can turned this int meaning 0=no data, 1=ZS data, 2=raw, 3=embediing...etc. Keeping it Int_t.